topics/spacevim/index.md

```diff
@@ -0,0 +1,16 @@
+---
+aliases: space vim
+created_by: ShiDong Wang
+display_name: SpaceVim
+github_url: https://github.com/SpaceVim/SpaceVim
+logo: spacevim.png
+released: November 6, 2017
+short_description: SpaceVim is a community-driven vim distribution that seeks to provide layer feature.
+topic: spacevim
+url: https://spacevim.org/
+---
+
+SpaceVim is a vim distribution inspired by spacemacs, SpaceVim manages collections
```

Comment:
You could link spacemacs if you like: `[spacemacs](https://github.com/syl20bnr/spacemacs)`

Can you split the sentence at the comma?

> SpaceVim is a vim distribution inspired by [spacemacs](https://github.com/syl20bnr/spacemacs). SpaceVim manages collections

It might be helpful to mention what vim is, for users new to it. Maybe in the first sentence:

> SpaceVim is a distribution of the vim editor that's inspired by

Reply:
Yeah, add link will be better. :)